tokio-console/Cargo.toml

```
@@ -29,14 +29,27 @@ keywords = [
 [dependencies]
 atty = "0.2"
 console-api = { version = "0.6.0", path = "../console-api", features = ["transport"] }
-clap = { version = "3", features = ["cargo", "derive", "env"] }
+# Use 1.64.0 compatible version of `clap@4.1.14`.
+clap = { version = "=4.1.14", features = [
+    # these are all enabled by default
+    "std", "color", "help", "usage", "error-context", "suggestions",
+    # auto-wrap help text to fit the terminal width
+    "wrap_help",
+    # use Cargo env vars when determining help text
+    "cargo",
+    # enable derive attributes
+    "derive",
+    # parse values from env vars
+    "env"
+] }
+clap_complete = "=4.1.6"
 tokio = { version = "1", features = ["full", "rt-multi-thread"] }
 tonic = { version = "0.10", features = ["transport"] }
 futures = "0.3"
 ratatui = { version = "0.20.1", default-features = false, features = ["crossterm"] }
 tower = "0.4.12"
 tracing = "0.1"
-tracing-subscriber = { version = "0.3.0", features = ["env-filter"] }
+tracing-subscriber = { version = "0.3.0" }
```

nit: we can collapse this down to the short version now that we're not pulling in non-default features:

```suggestion
tracing-subscriber = "0.3"
```

---

Fixed.